dhis-2/dhis-api/src/main/java/org/hisp/dhis/security/apikey/ApiTokenAttribute.java

```diff
@@ -45,13 +44,18 @@
     @JsonSubTypes.Type( value = RefererAllowedList.class, name = "RefererAllowedList" ),
     @JsonSubTypes.Type( value = MethodAllowedList.class, name = "MethodAllowedList" ) } )
 @JacksonXmlRootElement( localName = "apiTokenAttribute", namespace = DxfNamespaces.DXF_2_0 )
-public abstract class ApiTokenAttribute implements Serializable
+public abstract class ApiTokenAttribute implements EmbeddedObject
```

Comment:
why did you remove Serializable interface?

Reply:
As discussed, this change is not part of this PR.